tests/Mutant/Generator/MutationsGeneratorTest.php

```diff
@@ -61,15 +64,7 @@ final class MutationsGeneratorTest extends TestCase
 {
     public function test_it_collects_plus_mutation(): void
     {
-        $codeCoverageDataMock = $this->createMock(CodeCoverageData::class);
```

Comment:
Could you please replace (not remove) it with asserting `getAllTestsForMutation()` calls of `$codeCoverageDataMock`?

same below